RUBYGEM.md

```diff
@@ -0,0 +1,9 @@
+# Publish the Gem on RubyGems.org
+
+To publish the gem set `RUBYGEMS_EMAIL` and `RUBYGEMS_PASSWORD` environment variables with your email and password from your RubyGems account respectively.
+Then run the instructions to build and run the docker image to build and publish the gem.
```

Comment:
"Then run the following [Docker](https://docs.docker.com/engine/installation/) commands in the terminal to build and publish the gem."